src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -919,7 +945,7 @@ Unlike MySQL, PostgreSQL does support date time or timestamp values with an offs
 | Name         | Supported | Model Level | Field Level | Description |
 |--------------|:---------:|:-----------:|:-----------:|-------------|
 | `@model`     | ✅        | ✅          | ❌          | Creates a datasource and resolver for a table.            |
-| `@auth`      | ✅        | ✅          | ❌          | Allows access to data based on a set of authorization methods and operations. |
+| `@auth`      | ✅        | ✅          | ✅       | Allows access to data based on a set of authorization methods and operations. |
```

Comment:
Nit: it should be OK for markdown, just in case if you wanna keep the code formatting to follow the same spacing before the pipe `|` :P